Libraries/Image/Image.android.js

```diff
@@ -161,6 +162,11 @@ var Image = React.createClass({
       console.warn('source.uri should not be an empty string');
     }
 
+    // Provide a warning if using src instead of source
```

Comment:
remove the comment, it doesn't provide any information that's not obvious from reading the line :)